components/requesttask.brs

```diff
@@ -0,0 +1,19 @@
+function init()
+    m.top.functionName = "getcontent"
+end function
+
+function getcontent()
```

Comment:
This function is doing a lot of stuff. Can you split it into smaller functions?
https://refactoring.guru/smells/long-method